RFCs/FS-1126-allow-lower-case-du-cases when-require-qualified-access-is specified.md

```diff
@@ -28,6 +28,9 @@ type DU = | a // error FS0053: Discriminated union cases and exception labels mu
 
 type DU = | a // error FS0053: Discriminated union cases and exception labels must be uppercase identifiers
 ```
+type DU = | ``not.allowed`` // error FS0883: Invalid namespace, module, type or union case name
+
+Note: the above example is not a valid type name for .NET, so it will remain as an error.
 
 # Detailed design
 This RFC will avoid a compiler check for union case names if the `[<RequireQualifiedAccess>]` is used at the type level.
```

Comment:
Can we add a comment on whether or not, and why (if not), there will be a warning available (level 5) that informs users that lower-case DUs are being used? I can imagine that in general people would want to guard against this style of programming. Maybe this was discussed and dismissed, but then a line explaining that would be nice.

Reply:
This feature is currently in preview and user will see "Lowercase discriminated union cases are only allowed when using RequireQualifiedAccess attribute" . And when it is out of preview user will see the same error message reusing the same FS0053 in the case of when using lowercase without the use  use of RequireQualifiedAccess .